### PR TITLE
Website Redesign

### PR DESCRIPTION
**Pull request by itexpert120**

### CODE REVIEW

**Comment by itexpert120** [comment]
@elliotBraem Checkout this pr, I followed your comments from last pr. See if there are any changes needed

File: src/devhub/components/organism/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const imageLink =
```
Comment (elliotBraem) [inline src/devhub/components/organism/Banner.jsx:1]: You can remove this file, seems like a duplicate of island/Banner

File: src/devhub/components/templates/AppLayout.jsx [hunk 2]
```diff
@@ -35,78 +35,88 @@ const ContentContainer = styled.div`
   width: 100%;
 `;
```
Comment (elliotBraem) [inline src/devhub/components/templates/AppLayout.jsx:37]: Go ahead and just remove all the commented out; we don't need it anymore

File: src/devhub/components/molecule/NavbarDropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+const title = props.title;
```
Comment (elliotBraem) [inline src/devhub/components/molecule/NavbarDropdown.jsx:1]: You could remove this from molecule, we have it in organism

Comment (itexpert120) [reply src/devhub/components/molecule/NavbarDropdown.jsx:1]: resolved

**Pull request merged**